package.json

```diff
@@ -48,12 +48,21 @@
     "@commitlint/cli": "^17.6.5",
     "@commitlint/config-conventional": "^17.6.5",
     "@swc/core": "^1.3.66",
+    "assert": "^2.0.0",
```

Comment:
Why is it needed in the `root`? 
It's also repeated in the `demo/package.json`, so you can remove it from there.

Reply:
No, I can't. This is needed for running and building demo app. These deps are installed automatically because pnpm needs them explicitly.